src/main/java/nl/smallproject/www/techiteasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,48 @@
+package nl.smallproject.www.techiteasy.controllers;
+
+import nl.smallproject.www.techiteasy.models.Television;
+import nl.smallproject.www.techiteasy.services.TelevisionService;
+import org.apache.coyote.Response;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/televisions")
+public class TelevisionController {
+
+    private final TelevisionService televisionService;
```

Comment:
Je maakt al heel goed gebruik van de service. Zo hadden we dat in de les natuurlijk al gezien, maar het overstijgt deze opdracht.